apps/avifgainmaputil/tonemap_command.cc

```diff
@@ -0,0 +1,169 @@
+// Copyright 2023 Google LLC
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include "tonemap_command.h"
+
+#include <cmath>
+
+#include "avif/avif_cxx.h"
+#include "imageio.h"
+
+namespace avif {
+
+TonemapCommand::TonemapCommand()
+    : ProgramCommand("tonemap",
+                     "Tone maps an avif image that has a gain map to the "
+                     "requested headroom") {
```

Comment:
avifgainmaputil has no pedagogic goal but this is rather obscure for novices. Could there be some extra description for people discovering HDR?